client/app/account/billing/autoRenew/billing-autoRenew.controller.js

```diff
@@ -41,14 +41,15 @@ angular.module("Billing.controllers").controller("Billing.controllers.AutoRenew"
             text: $translate.instant("autorenew_service_type_ALL")
         };
 
-        let initLoading = true;
-
         $scope.loaded = false;
         $scope.user = null;
         $scope.expandHostingDomain = {};
         $scope.urls = {
             renewAlign: ""
         };
+        $scope.searchText = {
```

Comment:
Why do sometimes I have to bind `ngModel` to the property of an `Object` and sometimes I don't ?